widget/embedded/src/components/Layout/Layout.tsx

```diff
@@ -103,8 +106,8 @@ function Layout(props: PropsWithChildren<PropTypes>) {
         onClose={() => setShowActivateTabModal(false)}
         onConfirm={onActivateTab}
       />
-      {(hasLogo || footer) && (
-        <Footer>
+      <Footer>
+        {footer && (
```

Comment:
In previous code, footer was not rendered when hasLogo || footer was false, but it seems it's different now. 🤔 (also we have a similar case in modal)

Reply:
Yes, this is, We wanted to preserve this space as blank for the logo, so we changed the code.

Reply:
In previous code, when footer and hasLogo were both False, we didn't render anything. But now we return this: @mikasackermn @samobasquiat 

<Footer>
  <div
    className={`footer__logo footer__logo__hidden`}>
    <Divider size={12} />
    <BottomLogo />
  </div>
</Footer>

(previous result in this state):
<></>

Reply:
This can be accomplished in Layout, in my opinion, and we can identify hasLogo apart from the watermark.